src/Tribe/Body_Classes.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * Class used to manage and body classes across our plugins.
+ *
+ * @since TBD
+ */
+namespace Tribe;
+
+class Body_Classes {
+	/**
+	 * Stores all the classes.
+	 * In the format: ['class' => true, 'class => false ]
+	 *
+	 * @var array
+	 */
+	protected $classes = [];
+
+	/**
+	 * Register our methods in the correct places.
+	 *
+	 * @since TBD
+	 */
+	public function __construct() {
+		// Hook the actual adding of body classes.
```

Comment:
Missing the hook?

Reply:
Moved to the service provider - I'll just remove the empty `_construct`